bin/commands/support/verify-fingerprints/index.ts

```diff
@@ -0,0 +1,159 @@
+/*
+  This program and the accompanying materials are made available
+  under the terms of the Eclipse Public License v2.0 which
+  accompanies this distribution, and is available at
+  https://www.eclipse.org/legal/epl-v20.html
+
+  SPDX-License-Identifier: EPL-2.0
+
+  Copyright Contributors to the Zowe Project.
+*/
+
+import * as std from 'cm_std';
+import * as xplatform from 'xplatform';
+
+import * as common from '../../../libs/common';
+import * as fs from '../../../libs/fs';
+import * as javaCI from '../../../libs/java_ci';
+import * as shell from '../../../libs/shell';
+import * as stringlib from '../../../libs/string';
+
+function processCommResult(content: string, lines?: number): string {
+  let returnedOutput = '';
+  if (content) {
+    let linesSplit = content.split("\n");
+    if (lines && lines > 0) {
+      linesSplit = linesSplit.slice(0, lines);
+    }
+    linesSplit.forEach(line => {
+      const oneLineSplit = line.split(' ');
+      returnedOutput += `${oneLineSplit[0]}\n`
+    })
+  }
+  return returnedOutput;
+}
+
+export function execute(doNotExit: Boolean): void {
+
+  common.printLevel0Message('Verify Zowe file fingerprints');
+
```

Comment:
I got to the bottom of the build failures. The `require_java` from the shell script was removed so the dependency on a Zowe config file could be removed. Our tests used the Zowe config file and pulled JAVA_HOME from there...can we optionally re-implement that case when config exists?

i.e.

```typescript
  if (std.getEnv('ZWE_CLI_PARAMETER_CONFIG')) {
    requireJava()
  }
```